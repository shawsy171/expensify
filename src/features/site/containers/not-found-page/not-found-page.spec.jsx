import React from 'react';
import { shallow } from 'enzyme';

import NotFoundPage from './not-found-page';

describe('NotFoundPage Component', () => {
  test('should render correctly', () => {
    const wrapper = shallow(<NotFoundPage />);
    expect(wrapper).toMatchSnapshot();
  });
});

