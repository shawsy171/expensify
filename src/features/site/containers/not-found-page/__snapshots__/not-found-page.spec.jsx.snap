// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage Component should render correctly 1`] = `
<div>
  <p>
    404 Not found page
  </p>
  <Link
    replace={false}
    to="/"
  >
    Go Home
  </Link>
</div>
`;
